### PR TITLE
Use Fatalf instead of Errorf plus FailNow

### PR DESCRIPTION
**Pull request by YuxiJin-tobeyjin**
**What this PR does / why we need it**:
Use Fatalf instead of Errorf plus FailNow

**Release note**:
NONE
```release-note
```

### CODE REVIEW

**Comment by YuxiJin-tobeyjin** [comment]
ping @brendandburns @smarterclayton 
PTAL，thanks.

**Comment by YuxiJin-tobeyjin** [comment]
ping @brendandburns 
Maybe a typo,pls use /approve no-issue,thanks a lot.

**Comment by YuxiJin-tobeyjin** [comment]
ping @brendandburns 
Sorry to bother,PTAL
/approve no-issue

**Comment by YuxiJin-tobeyjin** [comment]
/assign @brendandburns
/approve no-issue
sir,pls

**Comment by YuxiJin-tobeyjin** [comment]
ping @brendandburns

File: pkg/kubectl/rolling_updater_test.go [hunk 1]
```diff
@@ -1516,13 +1516,11 @@ func TestUpdateRcWithRetries(t *testing.T) {
...
 	if err != nil {
-		t.Errorf("Error reading: %v", err)
-		t.FailNow()
+		t.Fatalf("Error reading: %v", err)
 	}
 	obj, err := runtime.Decode(codec, data)
 	if err != nil {
-		t.Errorf("error decoding: %v", err)
-		t.FailNow()
+		t.Fatalf("error decoding: %v", err)
```
Comment (xiangpengzhao) [inline pkg/kubectl/rolling_updater_test.go]: s/error/Error

**Review by xiangpengzhao — CHANGES_REQUESTED** [review]

**Comment by YuxiJin-tobeyjin** [comment]
Close

**Pull request closed without merging**